Libraries/LibWeb/Loader/ResourceLoader.cpp

```diff
@@ -575,9 +575,14 @@ void ResourceLoader::handle_network_response_headers(LoadRequest const& request,
     if (!request.page())
         return;
 
-    for (auto const& [header, value] : response_headers.headers()) {
-        if (header.equals_ignoring_ascii_case("Set-Cookie"sv)) {
-            store_response_cookies(*request.page(), request.url().value(), value);
+    if (request.respects_set_cookie()) {
```

Comment:
just wondering: why not to pass `include_credentials` enumeration down here, if the spec you cite explicitly mentions it?